src/webapp/forms.py

```diff
@@ -29,7 +29,7 @@ class GenerateDowntimeForm(FlaskForm):
     start_time = TimeField("&nbsp;", [InputRequired()])
     end_date = DateField("End Date/Time (UTC)", [InputRequired()])
     end_time = TimeField("&nbsp;", [InputRequired()])
-    services = SelectMultipleField("Services (select one or more)", [InputRequired()], choices=[])
+    services = SelectMultipleField("Known OSG Services (select one or more)", [InputRequired()], choices=[])
```

Comment:
Finally found a resource with multiple services, maybe we should change this to `(select one or more with ctrl-click)`?

Reply:
I would've put that but it's not ctrl-click on macs.